More content (#637)
temporalio/documentation: docs/content/what-is-an-activity-heartbeat.md

---
id: what-is-an-activity-heartbeat
title: What is an Activity Heartbeat?

> description: An Activity Heartbeat is ping, from the Worker that is executing the Activity to the Temporal Server, which serves as a way to ensure that the Activity Execution is making progress and the Worker has not crashed.

Reviewer: ```suggestion
description: An Activity Heartbeat is a ping from the Worker that is executing the Activity to the Temporal Server. Each ping informs the Temporal Server that the Activity Execution is making progress and the Worker has not crashed.
```
The second "ping" could be replaced by "Activity Heartbeat" to be more precise. Or we could drop the second sentence from the `description` field while leaving it in the body of the topic.

tags:
  - explanation
---

> An Activity Heartbeat is ping, from the Worker that is executing the Activity to the Temporal Server, which serves as a way to ensure that the Activity Execution is making progress and the Worker has not crashed.

Reviewer: ```suggestion
An Activity Heartbeat is a ping from the Worker that is executing the Activity to the Temporal Server.
Each ping informs the Temporal Server that the Activity Execution is making progress and the Worker has not crashed.
```
The second "ping" could be replaced by "Activity Heartbeat" to be more precise.
